.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+# The following configuration is used to setup GitHub Actions
+# for building (and running unit tests) on pushes and PRs.
+# At the moment, we build on two runner types: Linux and MacOS,
+# specific version information can be found further below.
+name: Build On Changes
+
+# run action on all pushes and pull_request for all branches
+on: [push, pull_request]
+
+# high-level constants
+env:
+  BASE_URL: https://gitlab.science.ru.nl/sac-group/sac-packages/-/raw/master/packages/weekly
+  VERS: 1.3.3-383-1
+  SPKG: sac2c-1.3.3-MijasCosta-383-g6ea12-omnibus
+
+# action listing, all jobs run in parallel
+jobs:
+  build_linux:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Get HEAD and submodules
+        uses: actions/checkout@v2
+        with:
+          # we need to get tag information
+          fetch-depth: 0
+          # we need to get all submodules
+          submodules: 'recursive'
+      - name: Install sac2c-basic
+        run: |
+          wget ${BASE_URL}/Ubl18/${VERS}/basic/${SPKG}.deb
+          sudo apt install ./${SPKG}.deb
+          sac2c -V
+      - name: Create build dir
+        run: |
+          cmake -E make_directory ${{runner.workspace}}/build
+          mkdir $HOME/.sac2crc
+      - name: Configure build-system
+        shell: bash
+        working-directory: ${{runner.workspace}}/build
+        run: cmake -DLINKSETSIZE=200 -DFULLTYPES=ON $GITHUB_WORKSPACE
+      - name: Build
+        shell: bash
+        working-directory: ${{runner.workspace}}/build
+        run: cmake --build . -j 2
+  build_mac:
+    runs-on: macos-10.15
+    steps:
+      - name: Get HEAD and submodules
+        uses: actions/checkout@v2
+        with:
+          # we need to get tag information
+          fetch-depth: 0
+          # we need to get all submodules
+          submodules: 'recursive'
+      - name: Install sac2c-basic
+        run: |
+          wget ${BASE_URL}/MacOS/${VERS}/basic/${SPKG}.pkg
+          sudo installer -pkg ./${SPKG}.pkg -target /
+          sac2c -V
+      - name: Create build dir
+        run: |
+          cmake -E make_directory ${{runner.workspace}}/build
+          mkdir $HOME/.sac2crc
```

Comment:
Same as above, isn't that done automatically by the sac2c build system.  If not, isn't that a bug?

Reply:
see reply above...